mods/ra/rules/infantry.yaml

```diff
@@ -388,6 +388,8 @@ DELPHI:
 
 CHAN:
 	Inherits: ^CivInfantry
+	Selectable:
+		Class: CHAN
```

Comment:
The point here is that inheritance overrides default `Class` attribution.
Here `^CivInfantry` already defines the selection class for `CHAN`, `GNRL` and other generic civilian actors, which they inherit. But `CHAN`, `GNRL` should not use this inherited `Class`, but should be different. See the example of the combat tank in D2K - the actual `Class` property is defined for only `^combat` but gets inherited by six different actors (3 for each race times 2 for normal / starport).